packages/@react-aria/selection/src/useSelectableCollection.ts

```diff
@@ -265,14 +266,22 @@ export function useSelectableCollection(options: SelectableCollectionOptions): S
     manager.setFocused(true);
 
     if (manager.focusedKey == null) {
+      let navigateToFirstKey = (key: Key | undefined, childFocus?: FocusStrategy) => {
```

Comment:
Is the `childFocus` bit prep for grid cases w/ focusable children? Just trying to figure out if useSelectableCollection will have access to childFocus information in onFocus

Reply:
so this function is actually stolen from above https://github.com/adobe/react-spectrum/blob/main/packages/@react-aria/selection/src/useSelectableCollection.ts#L114
it's a bit of copy pasta that i forgot about because lint didn't yell at me, it seems as though childFocus is only for arrow keys, so I'm thinking we don't need to worry about it at all